sections/header.liquid

```diff
@@ -926,6 +981,28 @@
       "default": "background-1",
       "label": "t:sections.all.colors.label"
     },
+    {
+      "type": "header",
+      "content": "t:sections.header.settings.header__3.content",
+      "info": "t:sections.header.settings.header__4.info"
```

Comment:
So weird that we're using two different headers.

Both the `info` and `content` should be under the same `header__3`.

But this is how it was implemented in the footer so I'm okay with keeping it as is.